### ISSUE TITLE
Mail form

### ISSUE THREAD

**Issue by Snajtz** [Sn8z/RottenTomato#7]
We should create a mail form of some kind for users to report problems, suggest features, suggest content etc

**Comment by Sn8z** [comment]
We also need to create a new email address :)

**Comment by Sn8z** [comment]
We should move the html code for the contact form to a partial! :dancer:

**Comment by Sn8z** [comment]
ping @Snajtz

**Comment by Snajtz** [comment]
You are completely correct, I will fix it :) Alternatively someone else can create the email if it's needed sooner.

**Comment by Sn8z** [comment]
This feature could be solved with a model instead, this way we could create some kind of interface for admins to read and handle stuff on the page itself.
